### PR TITLE
Add Base Goerli ChainSpec

### PR DESCRIPTION
**Pull request by N0xMare**

### CODE REVIEW

File: crates/primitives/src/chain/spec.rs [hunk 1]
```diff
@@ -267,6 +267,39 @@ pub static OP_GOERLI: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
...
+            (Hardfork::Petersburg, ForkCondition::Block(0)),
+            (Hardfork::Istanbul, ForkCondition::Block(0)),
+            (Hardfork::MuirGlacier, ForkCondition::Block(0)),
+            (Hardfork::Berlin, ForkCondition::Block(0)),
+            (Hardfork::London, ForkCondition::Block(0)),
+            (
+                Hardfork::Paris,
+                ForkCondition::TTD { fork_block: Some(0), total_difficulty: U256::from(0) },
+            ),
+            (Hardfork::Regolith, ForkCondition::Block(0)),
```
Comment (roberto-bayardo) [inline crates/primitives/src/chain/spec.rs:294]: I believe this should be [timestamp 1683219600](https://github.com/base-org/node/blob/8690d2890381087881464b344b64e4ca5ada04b8/goerli/rollup.json#L28)

Comment (N0xMare) [reply crates/primitives/src/chain/spec.rs:294]: did base genesis with regolith enabled?

Comment (refcell) [reply crates/primitives/src/chain/spec.rs:294]: Looks like it did based on the commit history comments of the base goerli rollup json file :)

Comment (refcell) [reply crates/primitives/src/chain/spec.rs:294]: @N0xMare is making the changes now.

Comment (N0xMare) [reply crates/primitives/src/chain/spec.rs:294]: just talked with clabby and refcell, changing now :)

**Review by clabby — APPROVED** [review]
LGTM

**Pull request merged**